pytest-fixture-config/pytest_fixture_config.py

```diff
@@ -40,11 +40,11 @@ def yield_requires_config(cfg, vars_):
     def decorator(f):
         # We need to specify 'request' in the args here to satisfy pytest's fixture logic
         @functools.wraps(f)
-        def wrapper(request, *args, **kwargs):
+        def wrapper(*args, **kwargs):
```

Comment:
This fails on Python 3 because (as I think) wrapped function doesn't accept `request` in test:
https://github.com/manahl/pytest-plugins/blame/master/pytest-fixture-config/tests/unit/test_fixture_config.py#L52

I'm not familiar with this part of pytest and I don't know how this can work on Python 2.
Just forwarding all arguments solved failure, plus this should have same semantic for correct code as before.

If my assumptions are correct `request` should be removed from `requires_config` too, but I'm unsure.
Please correct me if I'm wrong.